llms/mlx_lm/convert.py

```diff
@@ -69,7 +70,8 @@ def fetch_from_hub(
         weights.update(mx.load(wf).items())
 
     config = transformers.AutoConfig.from_pretrained(model_path)
-    tokenizer = transformers.AutoTokenizer.from_pretrained(model_path)
+    trust_remote_code = resolve_trust_remote_code(None, config.tokenizer_class, False, True)
```

Comment:
I think the new version of the transformer should fix the issue of not prompting the user to load a custom tokenizer, so we don't have to manually call `resolve_trust_remote_code`. Would you try installing the latest transforms as suggested in this link and see if it works without resolve_trust_remote_code? [Link](https://github.com/huggingface/transformers/issues/28501#issuecomment-1892237713)

Reply:
Thank you for pointing to the instruction, and I tried with the latest transformers library installed from github repo. Then, I confirmed that I can remove `resolve_trust_remote_code` from this code. I applied the modification in 6828f51, so I'd appreciate it if you would check this part again.